packages/presentation/lib/style/colors.dart

```diff
@@ -3,9 +3,11 @@ import 'package:flutter/material.dart';
 class MyColors {
   const MyColors._();
 
-  static int splashScreenUp = 0xffE51937;
+  static const int splashScreenUp = 0xffE51937;
 
-  static int splashScreenBottom = 0xffDB5252;
+  static const int splashScreenBottom = 0xffDB5252;
```

Comment:
why u add this variable with int and on 10 line without
if I understand u wand to use hex color? but other colors are RGBO, choose pls one type